.github/workflows/pr-build.yaml

```diff
@@ -0,0 +1,237 @@
+           echo $MAVEN_SETTINGS_XML > /home/runner/work/secrets/settings.xml
+
+        - name: Copy GPG passphrase
+          env:
+           GPG_PASSPHRASE: ${{ secrets.GPG_PASSPHRASE }}
+          run : |
+            echo $GPG_PASSPHRASE > /home/runner/work/secrets/passphrase.file
+
+        - name: Copy GPG key
+          env:
+            GPG_KEY_BASE64: ${{ secrets.GPG_KEY }}
+          run : |
+            echo $GPG_KEY_BASE64 |  base64 --decode > /home/runner/work/secrets/galasa.gpg
+            
+        - name: Make GPG home directory and change permissions
+          run: |
+            mkdir /home/runner/work/gpg
+            chmod '700' /home/runner/work/gpg
+
+        - name: Import GPG
+          run: |
+            gpg --homedir /home/runner/work/gpg --pinentry-mode loopback --passphrase-file /home/runner/work/secrets/passphrase.file --import /home/runner/work/secrets/galasa.gpg
+        
+        - name: Copy custom settings.xml
+          run: |
+            cp /home/runner/work/secrets/settings.xml /home/runner/work/gpg/settings.xml
+            
+        - name:  Pull and run the docker image
+          run: |
+             docker run --rm -v ${{github.workspace}}:/var/root/ ghcr.io/${{env.NAMESPACE}}/galasabld-amd64:${{env.IMAGE_TAG}} template --releaseMetadata var/root/framework/release.yaml --releaseMetadata /var/root/extensions/release.yaml --releaseMetadata /var/root/managers/release.yaml --releaseMetadata /var/root/obr/release.yaml --template /var/root/obr/javadocs/pom.template --output /var/root/obr/javadocs/pom.xml --javadoc
+             
+        - name: Display pom.xml 
+          run: |
+            cat ${{github.workspace}}/obr/javadocs/pom.xml
+          
+        - name: Build javadoc site using maven in directory javadocs
+          run: |
+            mvn -f ${{github.workspace}}/obr/javadocs/pom.xml deploy -Dgpg.skip=true \
+            -Dgalasa.source.repo=https://development.galasa.dev/gh/maven-repo/managers \
+            -Dgalasa.central.repo=https://repo.maven.apache.org/maven2/ \
+            -Dgalasa.release.repo=file:${{ github.workspace }}/obr/javadocs/docker/repo \
+            -Dmaven.javadoc.failOnError=false --batch-mode --errors --fail-at-end \
+            --settings  /home/runner/work/gpg/settings.xml
+        
+        - name: Build javadocsite Image for testing
+          uses: docker/build-push-action@v5
+          with:
+            context: .
+            file: ${{github.workspace}}/obr/dockerfiles/dockerfile.javadocsite
+            load: true
+            tags: javadocsite:test
+
+        - name: Build Javadoc Maven repo Image for testing
+          uses: docker/build-push-action@v5
+          with:
+            context: .
+            file: ${{github.workspace}}/obr/dockerfiles/dockerfile.javadocmavenrepo
+            load: true
+            tags: javadocmavenrepo:test
+            build-args: |
+              dockerRepository=${{env.REGISTRY}}
+              tag= latest
```

Comment:
Again please can we remove the space as I'm not sure if this will cause an issue
`tag=latest`

Reply:
done